.travis.yml

```diff
@@ -14,7 +14,7 @@ matrix:
 git:
   depth: 5
 env:
-  - "NODE_FLAGS='' SCRIPT_FLAGS=''"
+  - "NODE_FLAGS='--expose-gc' SCRIPT_FLAGS=''"
```

Comment:
To make Browserify happy, another option is to replace this commit with a `complete.exclude("v8");` inside browser_test_generator.js, if that's preferred.